pkg/doc/jose/encrypter_decrypter_test.go

```diff
@@ -159,6 +162,63 @@ func TestJWEEncryptRoundTrip(t *testing.T) {
 	}
 }
 
+func TestRoundTripInteropWithGoJose(t *testing.T) {
```

Comment:
what about Encrypt with our package and decrypt with go-jose?

Reply:
I'll add a test